pkgs/development/lua-modules/aliases.nix

```diff
@@ -42,4 +42,5 @@ mapAliases {
   lpty = throw "lpy was removed because broken and unmaintained "; # added 2023-10-14
   cyrussasl = throw "cyrussasl was removed because broken and unmaintained "; # added 2023-10-18
   nlua-nvim = throw "nlua-nvim was removed, use neodev-nvim instead"; # added 2023-12-16
+  nvim-client = throw "nvim-client was removed because broken and unmaintained"; # added 2023-12-17
```

Comment:
I would rather mention "removed because part of neovim" as stated on https://github.com/neovim/lua-client